domains/Accounts/routes.php

```diff
@@ -13,3 +16,21 @@
     'as' => 'users.verify',
     'uses' => VerifyEmailController::class,
 ]);
+
+Route::post('/login', [
```

Comment:
Should we let a request with a valid token hit this route?

Reply:
Are you proposing a middleware for when the user has an valid access_token redirecting to another route or deny access?

Reply:
Yeas, something along those lines. It doesn't make sense to execute a login operation if the user is already authenticated, no?

Reply:
Done!